manim/mobject/svg/text_mobject.py

```diff
@@ -591,22 +593,29 @@ def find_indexes(self, word: str, text: str):
     #         self.t2w = kwargs.pop("text2weight")
 
     def set_color_by_t2c(self, t2c=None):
-        """Internally used function. Sets color for specified strings."""
+        """Internally used function.
+
+        Sets color for specified strings.
+        """
         t2c = t2c if t2c else self.t2c
         for word, color in list(t2c.items()):
             for start, end in self.find_indexes(word, self.original_text):
                 self.chars[start:end].set_color(color)
 
     def set_color_by_t2g(self, t2g=None):
-        """Internally used. Sets gradient colors for specified
-        strings. Behaves similarly to ``set_color_by_t2c``."""
+        """Internally used.
+
+        Sets gradient colors for specified strings. Behaves similarly to
+        ``set_color_by_t2c``.
+        """
         t2g = t2g if t2g else self.t2g
         for word, gradient in list(t2g.items()):
             for start, end in self.find_indexes(word, self.original_text):
                 self.chars[start:end].set_color_by_gradient(*gradient)
 
     def text2hash(self):
         """Internally used function.
```

Comment:
All these internally used functions should be marked appropriately with a single underscore in front (perhaps in a future PR)